unit_testing/Util/serialStatus_tests.h

```diff
@@ -0,0 +1,41 @@
+/*!
+** @file   caBoardsUnitTests.h
+** @author Luke W
+** @date   09/08/2024
+*/
+
+#pragma once
```

Comment:
Should we stick to the `#ifndef/#define` rather than `#pragma once`?

Reply:
Yes - I was in a hurry when I was writing it, so used pragma, but I'd prefer to stick to #ifndef. I think its easier to manipulate.